resources/js/src/app/components/basket/list/BasketListItem.js

```diff
@@ -28,6 +29,36 @@ Vue.component("basket-list-item", {
 
     computed:
     {
+        activeOrderProperties()
+        {
+            const data = [];
+
+            for (const orderParam of this.basketItem.basketItemOrderParams)
+            {
+                const property = this.basketItem.variation.data.properties.find(property => property.property.id === parseInt(orderParam.propertyId));
+                const position = property && property.property && property.property.position;
+
+                data.push({
+                    ...clone(orderParam),
```

Comment:
Warum muss das Objekt hier geklont werden? Wie viel macht das Einbinden von lodash in der finalen JS-Datei aus. Brauchen wirr lodash hier wirklich?